cmake/toolchain/arcmwdt/generic.cmake

```diff
@@ -0,0 +1,22 @@
+# SPDX-License-Identifier: Apache-2.0
+
+set_ifndef(MWDT_TOOLCHAIN_PATH "$ENV{MWDT_TOOLCHAIN_PATH}")
```

Comment:
I would suggest that because the toolchain variant is named: `arcmwdt`, then use the name: `ARCMWDT_TOOLCHAIN_PATH`

That would be aligned with:
`gnuarmemb`: `GNUARMEMB_TOOLCHAIN_PATH`
`espressif`: `ESPRESSIF_TOOLCHAIN_PATH`
`xtools`: `XTOOLS_TOOLCHAIN_PATH`

Reply:
Nice catch, fixed